pkgs/games/hex-a-hop/default.nix

```diff
@@ -0,0 +1,41 @@
+{lib, stdenv, makeDesktopItem, copyDesktopItems, fetchurl, SDL, SDL_mixer, SDL_ttf}:
+
+stdenv.mkDerivation rec {
+   pname = "hex-a-hop";
+   version = "1.1.0";
+
+   src = fetchurl {
```

Comment:
```suggestion
   src = fetchzip {
```

Reply:
You need to update the hash aswell.

Reply:
And the import, done in 81724c9c8625f0ee91d93a4b19ee3bbbee56a96a